### DISCUSSION TITLE
Lesson Notes for Course 1

### DISCUSSION THREAD

**Discussion by nahuakang** [OpenMined/courses#137] [Resources]
I shared my notes for the first two lessons around and some people have shown interest in them. So I'll continue writing notes for the entire Course 1 and keep updating them in the following 2 channels:

- Blog: https://www.nahua.dev/series/privacy_ai_course_notes/
- Gdrive (commenter access given): https://drive.google.com/drive/folders/1n7JvNny7KAFiYDtlInvZpoQvfZNYfeoF?usp=sharing

If you want to help out with the notes in Gdrive, feel free to ask for editing access.

**Note:** I work full-time and have other projects that I contribute to on a daily basis, so it might take me some time to finish Course 1.

**Comment by souravcipher** [comment]
Loved the notes @nahuakang!

I'm going through it and I'm sure people who are taking the course right now as well in the future will find it very helpful, Amazing work. 🎉
